### PR TITLE
Release 0.17.0

### PR DESCRIPTION
**Pull request by robinlinden**
I think I put everything relevant in the changelog, but it'd be nice if someone could check that I got everything.

---
This change is [Reviewable](https://reviewable.io/reviews/utox/utox/1200)

### CODE REVIEW

**Comment by robinlinden** [comment]
@uTox/i18n Anyone up for translating a new release?

**Comment by redmanmale** [comment]
Changelog traslation into Russian.
https://gist.github.com/redmanmale/b1490eeb62a919679668cd42b2175264

---

Reviewed 4 of 4 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---

*[src/branding.h, line 20 at r1](https://beta.reviewable.io/reviews/utox/utox/1200#-L9_Hty79WtjeAERwU0P:-L9_Hty79WtjeAERwU0Q:b-80zug1) ([raw file](https://github.com/utox/utox/blob/74d1fcfe9cf97c9299e79ca2f80015788892852f/src/branding.h#L20)):*
> ```objc
> // Assembly info
> #define UTOX_FILE_DESCRIPTION "The lightweight Tox client"
> #define UTOX_COPYRIGHT "Copyleft 2017 uTox contributors. Some rights reserved."
> ```

Update year.

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/utox/utox/1200#-:-L9_Jslg28SjknG5XPFE:b-nv0fiv)*

**Comment by robinlinden** [comment]
Review status: 2 of 5 files reviewed at latest revision, 1 unresolved discussion.

---

*[src/branding.h, line 20 at r1](https://beta.reviewable.io/reviews/utox/utox/1200#-L9_Hty79WtjeAERwU0P:-L9_Nb6F1h6Z8tb5xIuX:b-896fix) ([raw file](https://github.com/utox/utox/blob/74d1fcfe9cf97c9299e79ca2f80015788892852f/src/branding.h#L20)):*
<details><summary><i>Previously, redmanmale (redmanmale) wrote…</i></summary><blockquote>

Update year.
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/utox/utox/1200)*

File: CMakeLists.txt [hunk 1]
```diff
@@ -7,11 +7,11 @@ project(uTox LANGUAGES C)
...
 
 set(PROJECT_VERSION_MAJOR "0")
-set(PROJECT_VERSION_MINOR "16")
-set(PROJECT_VERSION_PATCH "1")
+set(PROJECT_VERSION_MINOR "17")
+set(PROJECT_VERSION_PATCH "0")
 set(PROJECT_VERSION "${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}")
 
-set(PROJECT_COPYRIGHT "Copyleft 2017 uTox contributors. Some rights reserved.")
+set(PROJECT_COPYRIGHT "Copyleft 2018 uTox contributors. Some rights reserved.")
```
Comment (GrayHatter) [inline CMakeLists.txt]: This should say copyright, it's a legal claim, copyleft means nothing, and it's not really copyleft if we're licensing MIT now.

**Comment by GrayHatter** [comment]
:lgtm:

File: CMakeLists.txt (hunk 1 above)
Comment (redmanmale) [reply CMakeLists.txt]: Than fix it or drop it.

**Comment by endoffile78** [comment]
:lgtm:

---

Reviewed 2 of 4 files at r1, 3 of 3 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/utox/utox/1200#-:-L9bs5pY9ipuKFcCT8DR:bnfp4nl)*

**Comment by redmanmale** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

*[langs/uk.h, line 507 at r3](https://beta.reviewable.io/reviews/utox/utox/1200#-L9uf-UOCDFt_4ULWvTg:-L9uf-UOCDFt_4ULWvTh:b-7l1e2c) ([raw file](https://github.com/utox/utox/blob/6bf72e914590a776bed6df3723ac0d7a6c463d1c/langs/uk.h#L507)):*
> ```objc
> 
> msgid(STATUS_OFFLINE)
> msgstr("Offline")
> ```

Remove or translate please.

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/utox/utox/1200)*

**Comment by redmanmale** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/utox/utox/1200#-:-LAFRk370VYSH6Xjo60I:bnfp4nl)*

**Comment by endoffile78** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r4.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/utox/utox/1200#-:-LAP84Ul15Fg_vA6_HTR:bnfp4nl)*

**Comment by robinlinden** [comment]
Review status: all files reviewed at latest revision, all discussions resolved, some commit checks failed.

---

*[CMakeLists.txt, line 14 at r2](https://beta.reviewable.io/reviews/utox/utox/1200#-L9_4sUHKThquRxvwglE-r2-14:-LAPT3q3-n_qKqBK-mEh:b3uwm18) ([raw file](https://github.com/utox/utox/blob/c4cc2070ae92b0092fc69b9669d4de7389010740/CMakeLists.txt#L14)):*
<details><summary><i>Previously, redmanmale (redmanmale) wrote…</i></summary><blockquote>

Than fix it or drop it.
</blockquote></details>

Should've been done when the licence was changed then and it's not something I want to start messing with in the release PR. I'll open a separate PR changing it.

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/utox/utox/1200)*

**Pull request merged**